packages/chart/src/components/EditorPanel/EditorPanel.tsx

```diff
@@ -776,9 +777,27 @@ const EditorPanel = () => {
     return Object.keys(columns)
   }
 
-  const getDataValueOptions = data => {
+  const getLegendStyloptions = option => {
```

Comment:
Review spelling and casing getLegendStyleOptions

Reply:
Fixed